LootLockerServerSDK/Source/LootLockerServerSDK/Public/LootLockerServerForBlueprints.h

```diff
@@ -937,4 +939,177 @@ class LOOTLOCKERSERVERSDK_API ULootLockerServerForBlueprints : public UObject
      */
     UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Purchases", meta = (DeprecatedFunction, DeprecationMessage = "This method has been deprecated. We will soon release an updated purchase system, so please refrain from using this method."))
     static void CheckPurchaseStatusForPlayerByPlatformTransactionID_IncludeProducts(int PlayerID, int PlatformTransactionID, const FLootLockerServerPurchaseStatusWithProductsResponseBP& OnCompletedRequest);
+
+    //==================================================
+    // Player Progressions https://ref.lootlocker.com/server-api/#player-progressions
+    //==================================================
+
+    /**
+     * Get a list of progressions for the specified player
+     * https://ref.lootlocker.com/server-api/#get-player-progressions
+     *
+     * @param PlayerID ID of the player for whom to get progressions
+     * @param OnCompletedRequest Delegate for handling the server response
+     */
+    UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Player Progressions")
+    static void GetProgressionsForPlayer(int PlayerID, const FLootLockerServerPlayerProgressionListResponseBP& OnCompletedRequest);
```

Comment:
I suggest adding something about how many entries will be returned from this, since we also have the paginated functions. So something like; "This will return max 100 (or how many it is)", if you want to get more progressions use the paginated version of this function" or something like that.